pkg/storage/task.go

```diff
@@ -0,0 +1,198 @@
+package storage
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+// AddTask добавляет задачу в БД
+func (s *TaskStore) AddTask(task Task) (int64, error) {
+	res, err := s.db.Exec("INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)",
+		task.Date,
+		task.Title,
+		task.Comment,
+		task.Repeat,
+	)
+	if err != nil {
+		return 0, fmt.Errorf("Add task failed: %w", err)
+	}
+	return res.LastInsertId()
+}
+
+// GetTasks возвращает список задач
+func (s *TaskStore) GetTasks(limit int) ([]*Task, error) {
+	var tasks []*Task
+
+	query := "SELECT id, date, title, comment, repeat FROM scheduler ORDER BY date ASC LIMIT ?"
+	rows, err := s.db.Query(query, limit)
+	if err != nil {
+		return nil, fmt.Errorf("Get tasks failed: %w", err)
+	}
+	defer rows.Close()
+
+	for rows.Next() {
+		var task Task
+		if err := rows.Scan(
+			&task.ID,
+			&task.Date,
+			&task.Title,
+			&task.Comment,
+			&task.Repeat,
+		); err != nil {
+			return nil, fmt.Errorf("Failed to scan task: %w", err)
+		}
+		tasks = append(tasks, &task)
+	}
+
+	if err := rows.Err(); err != nil {
+		return nil, fmt.Errorf("Get tasks failed: %w", err)
```

Comment:
Лучше с маленькой писать ошибки при оборачивании
```suggestion
		return nil, fmt.Errorf("get tasks failed: %w", err)
```
![image](https://github.com/user-attachments/assets/1cada70e-42ae-48bb-aa9d-16f3e9fd4563)